### ISSUE TITLE
Syntax Error with SDXL on v21.8.7 (Lora Training)

### ISSUE THREAD

**Issue by alkah3st** [bmaltais/kohya_ss#1383]
I'm running into what appears to be a syntax error in the latest version (21.8.7)

I'm trying to use the Lora training with SDXL. My settings match that of what's suggested in the video on the tutorial of this repo. Torch is v2 and I'm on Python 3.10.9. My gpu is a 3090.

Tried pasting the error but markdown makes it unreadable. Instead, here's a link to the console:

https://imgur.com/a/0OsNQV7

 (Weirdly imgur is marking this as "erotic"--I guess the syntax error has aroused the server...)

EDIT: It looks like this is caused if you don't have gradient checkpointing checked. The "syntax errors" are apparently entirely irrelevant. Once I turned that on (which is mentioned after the parameters are discussed in the video) the training kicked off!
